shm/cargo.cpp

```diff
@@ -1,11 +1,31 @@
 #include "cargo.hpp"
 
-Cargo& Cargo::operator+=(size_t amount) {
+Cargo::Cargo(uint32_t amount, std::string name, uint32_t basePrice)
+    : amount_(amount), name_(name), basePrice_(basePrice) {}
+Cargo::~Cargo() {}
+
+uint32_t Cargo::getAmount() const {
+    return amount_;
+}
+
+uint32_t Cargo::getBasePrice() const {
+    return basePrice_;
+}
+
+std::string Cargo::getName() const {
+    return name_;
+}
+
+Cargo& Cargo::operator+=(uint32_t amount) {
     amount_ += amount;
     return *this;
 }
 
-Cargo& Cargo::operator-=(size_t amount) {
+Cargo& Cargo::operator-=(uint32_t amount) {
     amount_ -= amount;
     return *this;
 }
```

Comment:
Are we going to add some conditions, i.e. for += that amount should not be higher than ship capacity?

Reply:
Yes, it is needed, but I think that we will do it in SHM#2 because we have to know class relationships to use capacity in Cargo (capacity is in Ship)

Reply:
It is ok for me, should I open issue to remember about that?

Reply:
Probably not needed